core/components/clientconfig/lexicon/pt-br/default.inc.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * ClientConfig
+ *
+ * Copyright 2011-2014 by Mark Hamstra <hello@markhamstra.com>
+ *
+ * ClientConfig is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU General Public License as published by the Free Software
+ * Foundation; either version 2 of the License, or (at your option) any later
+ * version.
+ *
+ * ClientConfig is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+ * A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * ClientConfig; if not, write to the Free Software Foundation, Inc., 59 Temple
+ * Place, Suite 330, Boston, MA 02111-1307 USA
+ *
+ * @package clientconfig
+ * @subpackage lexicon
+ * @language pt (Translated by João Nogueira - JANogueira)
+ * 
+ */
+
+include_once(dirname(dirname(__FILE__)).'/en/default.inc.php'); // fallback for missing defaults or new additions

```

Comment:
Please remove this - it is not necessary. MODX automatically falls back to English lexicons when they're not found in the loaded language.

Reply:
Done